### ISSUE TITLE
vod.move not working as expected

### ISSUE THREAD

**Issue by 4syno4** [MrBrax/LiveStreamDVR#394]
Hi

I understand the archive button moves videos from vods folder to saved_vods, but for whatever reason it is not working in my set up.
Pressing the button simply does nothing. It asks if you want the video moved, but no result. No error log. not sign of the software failing. Nothing. I don't think my set up is that much different from vanilia, so I don't know why this is the case. Please help

**Comment by MrBrax** [comment]
> 2022-12-19 19:00:43 | vod.move | INFO | Move capturevid to /usr/local/share/twitchautomator/data/storage/saved_vods

so this is the only output? what about in the server console log?

**Comment by 4syno4** [comment]
![Screenshot_20221220_100021_Firefox](https://user-images.githubusercontent.com/106082667/208557258-730d6480-a799-4fb2-beb1-b8f104b9aa3b.jpg)
About the same. No error logs

**Comment by MrBrax** [comment]
hmm i don't have many leads on this, only other thing i can think of is that it doesn't reach the move function because the file doesn't exist

**Comment by 4syno4** [comment]
Yes same here. But I have to say the files are there. I just ended up wiping the docker and reinstalling.
